dts/bindings/mmc/st,stm32-sdmmc.yaml

```diff
@@ -29,3 +29,14 @@ properties:
         type: phandle-array
         required: false
         description: Power pin
+
+    bus-width:
+        type: int
+        required: false
+        default: 1
+        description: |
+            bus width for SDMMC access, defaults to 1
```

Comment:
Please justify why you have chosen the value 1, instead of simply saying that the default is 1. The default value is already visible on the line above and will show up in the bindings index HTML pages already. The reason why defaults are chosen must be given as a matter of policy in new Zephyr bindings.

Reply:
done